Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml

```
@@ -844,19 +907,227 @@ script:
           description: List of recipients included in the rule.
           type: Unknown
         - contextPath: O365Defender.SafeLinks.Rule.SentToMemberOf
-          description: List of distribution groups, dynamic distribution groups, or mail-enabled security groups included in the rule.
+          description: List of distribution groups, dynamic distribution groups, or
+            mail-enabled security groups included in the rule.
           type: Unknown
         - contextPath: O365Defender.SafeLinks.Rule.State
           description: The state of the rule.
           type: String
         - contextPath: O365Defender.SafeLinks.Rule.WhenChanged
-          description: 'The date and time the rule was modified. Time format: YYYY-MM-DDThh:mm:ss+00:00.'
+          description: "The date and time the rule was modified. Time format:
+            YYYY-MM-DDThh:mm:ss+00:00."
+          type: Date
+    - name: o365-defender-safelinks-detailed-report-get
+      description: Get detailed information about Safe Links results for the last 7
+        days. Yesterday is the most recent date that you can specify.
+      deprecated: false
+      arguments:
+        - name: start_date
+          description: Start date of the date range in MM-DD-YYYY format.Yesterday is the
```

```suggestion
          description: Start date of the date range in MM-DD-YYYY format. Yesterday is the
``` 
Space here, for the other cases too